docs/index.asciidoc

```diff
@@ -27,4 +27,6 @@ include::./distributed-tracing-guide.asciidoc[]
 
 include::./opentracing.asciidoc[]
 
-include::./release-notes.asciidoc[]
\ No newline at end of file
+include::./release-notes.asciidoc[]
+
+include::./framework-integrations.asciidoc[]
```

Comment:
I would like to see this moved elsewhere in the docs. I don't think at the end makes sense. Perhaps after configuraiton?